src/button/button-group.scss

```diff
@@ -20,7 +21,7 @@
       border-bottom-right-radius: $iui-border-radius;
     }
 
-    &:not(:last-of-type) {
+    &:not(.iui-borderless):not(.iui-cta):not(.iui-high-visibility):not(:last-of-type) {
```

Comment:
wow..

Reply:
I'm trying to target the default button, but the default button doesn't have a modifier class like borderless, CTA, and high visibility do.  I'm not sure how else I could pull this off?

Reply:
It shows that we might need to have `iui-default` class on our components...

Reply:
Yup. Button refactor needs to reduce specificity by adding two new classes: `.iui-default` (modifier) and `.iui-button-icon`.

Reply:
Where this change is required?

Reply:
Leaving for now, but creating item to get rid of it with the next buttons PR..